content/schedule/registration.md

```diff
@@ -14,5 +14,11 @@ During registration all team members and on-site coaches need to register. You w
 ## Location
 The registration takes place in [Faculty EEMCS](https://map.tudelftcampus.nl/poi/elektrotechniek-wiskunde-en-informatica-ewi/).
 This is the tall red/blue building.
-Exact location will be added later.
-For directions, see the [Location page]({{< relref "location" >}} "Location").
+
+The registration is held in the `\` (backslash) in the basement of the Faculty EEMCs.
+Follow the signs, or use the following directions. 
+When entering the building, walk towards the elevators and take the stairs down on the right hand side.
+At the bottom of the stairs go right through two red sliding doors.
+Walk around the green cage and turn right. 
```

Comment:
I believe this instruction ends at the entrance of the /Pub, should we mention that the Back/ is across the pub?

Reply:
I do indeed think around the cage goes directly to back/, but not 100 on that. I can go look later today, or we take a safe option and say "follow the signage" after the direction to walk through the two red doors

Reply:
Oohh, I wasn't aware of that shortcut, yes that would be nice to check :smile:

Reply:
ok, i hope this commits clarifies it.

Reply:
> I do indeed think around the cage goes directly to back/, but not 100 on that. I can go look later today, or we take a safe option and say "follow the signage" after the direction to walk through the two red doors

Thats why i started with "follow the signs" :P